CMakeLists.txt

```diff
@@ -451,33 +441,69 @@ if(MSVC)
   )
 endif()
 
+#
+# Find boost
+if(BUILD_WITH_BOOST)
+  hunter_add_package(Boost)
+  find_package(Boost CONFIG REQUIRED)
+  target_link_libraries(${LIB_NAME} Boost::boost)
+endif()
+
+#
+# Find Expat
+if(ARABICA_XML_BACKEND STREQUAL USE_EXPAT)
+  hunter_add_package(Expat)
+  find_package(EXPAT CONFIG REQUIRED)
+  target_link_libraries(${LIB_NAME} EXPAT::expat)
+endif()
 
 #
-# Add build dependency to boost if boost is part of the same project
-if(TARGET boost)
-  add_dependencies(${LIB_NAME} boost)
+# find libxml2
+if(ARABICA_XML_BACKEND STREQUAL USE_LIBXML2)
+  hunter_add_package(libxml2)
+  find_package(libxml2 CONFIG REQUIRED)
+  target_link_libraries(${LIB_NAME} ${LIBXML2_LIBRARIES})

```

Comment:
Please use the imported target:
- https://docs.hunter.sh/en/latest/packages/pkg/libxml2.html